src/angular/hq/src/app/staff-admin/staff-admin-list/staff-admin-list.component.html

```diff
@@ -0,0 +1,188 @@
+<div class="px-5">
```

Comment:
We should use the new list implementation with hq-table component for this new staff admin list page (implement StaffAdminListService, add hq-table and hq-table-body, etc.